config/auth0.php

```diff
@@ -7,6 +7,8 @@
  * https://github.com/auth0/auth0-PHP#configuration-options.
  */
 return [
+    'enabled' => env('AUTH0_INTEGRATION_ENABLED', false),
```

Comment:
This can be confusing because the Auth0 login (also configured in this file) is not enabled/disabled by this (or at least should not be)